syntax = "proto3";

import "google/protobuf/any.proto";

package proto.db;

message request {
	uint64 session_id = 1;
	uint32 associate_id = 2;
	uint32 type = 3;
	google.protobuf.Any content = 4;
}